src/main/java/ch/njol/skript/expressions/ExprTargetedBlock.java

```diff
@@ -36,55 +37,69 @@
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Targeted Block")
-@Description("The block at the crosshair. This regards all blocks that are not air as fully solid, e.g. torches will be like a solid stone block for this expression.")
-@Examples({"# A command to set the block a player looks at to a specific type:",
-		"command /setblock &lt;material&gt;:",
-		"\ttrigger:",
-		"\t\tset targeted block to argument"})
-@Since("1.0")
+@Description({
+	"The block at the crosshair. This regards all blocks that are not air as fully solid, e.g. torches will be like a solid stone block for this expression.",
+	"The actual target block will regard the actual hit box of the block."
+})
+@Examples({
+	"set target block of player to stone",
+	"set target block of player to oak_stairs[waterlogged=true]",
+	"break target block of player using player's tool",
+	"give player 1 of type of target block",
+	"teleport player to location above target block",
+	"kill all entities in radius 3 around target block of player",
+	"set {_block} to actual target block of player",
+	"break actual target block of player"
+})
+@Since("1.0, INSERT VERSION (implement 'actual/exact' correctly)")
 public class ExprTargetedBlock extends PropertyExpression<Player, Block> {
 
 	static {
 		Skript.registerExpression(ExprTargetedBlock.class, Block.class, ExpressionType.COMBINED,
-				"[the] target[ed] block[s] [of %players%]", "%players%'[s] target[ed] block[s]",
-				"[the] actual[ly] target[ed] block[s] [of %players%]", "%players%'[s] actual[ly] target[ed] block[s]");
+				"[the] [actual:(actual[ly]|exact)] target[ed] block[s] [of %players%]", "%players%'[s] [actual:(actual[ly]|exact)] target[ed] block[s]");
```

Comment:
Little concerned about `actually` being a valid pattern, I can't see any way it makes sense when used.
```
the actually target block of player
player's actually targeted block
```

Reply:
That was the original pattern, its bad but really no point to remove it and create a breaking change.

Reply:
Looking at this again I think it's valid to remove it, the `actual[ly]` pattern was already deprecated with a warning this was done in 2021. If the plan is to remove the deprecation then I believe removing `ly` should be fine. Still would be marked as breaking change

Reply:
it could go either way
I don't think there's much harm in keeping it or much of a deal about removing it

Reply:
alright then, feel free to mark it as resolved since for some reason I can't do it